### ISSUE TITLE
Color contrast of Apply | Give | Contact is not accessible

### ISSUE THREAD

**Issue by jeremyfelt** [washingtonstateuniversity/education.wsu.edu#2]
These are getting flagged as having a contrast ratio of 3.46:1
